translations/German/general-notes.md

```diff
@@ -0,0 +1,82 @@
+# General Rules
+
+Thank you for considering to contribute to the German translation!
+Before you start writing, please make sure that you have read the
+following general translation rules.
+
+## Formal or Informal?
+
+**Informal.**
+
+Although written German tends to be quite formal, websites in German are
+usually following informal netiquette. As Zulip's guides are written
+in a more colloquial style, German translations should be rather informal as well.
+
+**Don't use slang or regional phrases in the German translation:**
+* ~~"So'n Dreck kann jedem mal passieren."~~
+* ~~"Das ist die Seite, wo der Quelltext steht."~~
+
+## Form of address
+
+**Use "Du" instead of "Sie".**
+
+For the reasons provided in #Formal_or_Informal, stick to *Du*(informal)
+instead of *Sie*(formal) when addressing the reader.
+Always capitalize the *Du* when addressing him, do **not** use *du*.
+
+## Form of instruction
+
+**Prefer imperative over constructions with auxiliary verbs.**
+
+For instructions, try to use the imperative (e.g. *"Gehe auf die Seite"* -
+*"Go to the page"*) instead of constructions with auxiliary verbs
+(e.g. *"Du musst auf die Seite ... gehen"* - *"You have to go the page ..."*).
+This keeps the phrases short and avoids unnecessary addressings of the reader.
+
+## Form of titles
```

Comment:
Yes, `titles` is a little ambiguous. Imo  `form of verbs` doesn't fit a 100% either, since this is referring to the text on buttons, menu-items, etc. What do you think of `Rules for captions`?